### DISCUSSION TITLE
Confused about pricing

### DISCUSSION THREAD

**Discussion by WakanaYuki** [plausible/analytics#983] [Self-Hosted Support] [answered]
Hey,

so I set up a self-hosted instanced of plausible yesterday.
I got everything up and running and created the stat site for my website.
Now I got an email (from my self-hosted instance) saying it activated a free 30-day unlimited use trial.
Does that mean I can only use my self-hosted instance for the next 30 days and then need to pay up?
Or is self-hosted plausible free forever? Like with self-hosted Bitwarden_rs, Sentry, etc. etc.
Or do I need to pay a license cost to continue running it self-hosted on my server?

**Comment by Vigasaurus** [comment, marked as answer]
Self hosted is free forever. See https://plausible.io/docs/self-hosting#3-start-the-server - in the self hosted setup, it: 
> Creates an admin account (which is just a normal account with a generous 100 years of free trial)
